.github/workflows/build-project.yaml

```diff
@@ -28,25 +29,30 @@ jobs:
 
           case "${GITHUB_EVENT_NAME}" in
             pull_request)
-              config_data=('codesign:false' 'notarize:false' 'package:false' 'config:RelWithDebInfo')
-              if gh pr view ${{ github.event.number }} --json labels \
+              config_data=('codesign:false' 'notarize:false' 'package:false' 'config:RelWithDebInfo' 'flatpakArch:["x86_64"]')
+              label_data=$(gh pr view ${{ github.event.number }} --json labels)
+              if echo "${label_data}" \
                 | jq -e -r '.labels[] | select(.name == "Seeking Testers")' > /dev/null; then
                 config_data[0]='codesign:true'
                 config_data[2]='package:true'
               fi
+              if echo "${label_data}" \
```

Comment:
I'm not really happy about introducing a bespoke label just for this, but code-wise it's fine.

Reply:
I could work and make the aarch64 build happen on PR if a cache is there and a label to force the build without cache.
But I would prefer to know, how well/fast it is on master before enabling on every PR (if cache available).

Note that the label name can be discussed.